File: api/src/main/java/com/leandroadal/vortasks/entities/social/tasks/enumerators/Difficulty.java
package com.leandroadal.vortasks.entities.social.tasks.enumerators;

public enum Difficulty {
    EASY("Fácil"),
    MEDIUM("Médio"),
    HARD("Difícil"),
    VERY_HARD("Muito Difícil");

    private final String description;

    Difficulty(String description) {
        this.description = description;
    }

    public String getDescription() {
        return description;
    }
}

File: app/vortasks/lib/models/enums/difficulty.dart
// ignore_for_file: constant_identifier_names

enum Difficulty {
  EASY('Fácil'),
  MEDIUM('Médio'),
  HARD('Difícil'),
  VERY_HARD('Muito Difícil');

  final String namePtBr;

  const Difficulty(this.namePtBr);
}
